libraries/js/src/documents/loader.ts

```diff
@@ -0,0 +1,115 @@
+import { credentials, dataIntegrity, multikey, undefinedTerms } from './static.js';
+import { isObj } from '../types/general.js';
+import { getCached, setCached } from './cache.js';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type DidDoc = Record<string, any>;
+
+const documents = new Map<string, DidDoc>();
+
+documents.set(dataIntegrity.id, dataIntegrity.doc);
+documents.set(credentials.id, credentials.doc);
+documents.set(undefinedTerms.id, undefinedTerms.doc);
+documents.set(multikey.id, multikey.doc);
+
+// This builds a did:key version of the document
+function keyToDidDocument(url: string, key: string) {
+  const rootUrl = url.split('#')[0];
+  return {
+    '@context': ['https://www.w3.org/ns/did/v1', 'https://w3id.org/security/multikey/v1'],
```

Comment:
Both will work; contexts apply to the scope that they are directly enclosed in as well as all child scopes. I think in this case because the key is sometimes served as a standalone "document" (via the fragment URL) it needs to be in the child scope, and if you do that, you don't also need to do it at the global level. But there's also nothing wrong with re-declaring it as you have it here.. more of a stylistic preference than anything else.